LLM/OSS/Open_AI_OSS.py

```
@@ -23,11 +25,58 @@
+                connect_timeout=3,
+            )
+        with conn:
+            with conn.cursor() as cur:
+                cur.execute(
+                    """
+                    INSERT INTO chatbot_logs (query, mode, response, url, cache_hit, latency_ms)
+                    VALUES (%s, %s, %s, %s, %s, %s)
+                    """,
+                    (query, mode, response, url, cache_hit, latency_ms),
+                )
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)